### DISCUSSION TITLE
Should `<Script>` be used inside `<Head>`

### DISCUSSION THREAD

**Discussion by gvocale** [vercel/next.js#36358] [Help]
When loading a Google Analytics script tag via Next's `<Script>`, as shown in the example here https://nextjs.org/docs/messages/next-script-for-ga, should it be wrapped with Next `<Head>` or not?

Would wrapping `<Script>` with `<Head>` in `_app.tsx` have any counter effect?

Thanks!

**Comment by vadymshymko** [comment]
in short, the answer is no. More [here](https://nextjs.org/docs/basic-features/script)
